### ISSUE TITLE
npm install failing

### ISSUE THREAD

**Issue by dgroh** [highcharts/highcharts-editor#214]
#### Expected behaviour
I want to `npm install` the cloned repository.

I use node **12.16.1** and npm **6.13.4** in Windows:

#### Actual behaviour
When executing `npm install` from the cloned repository I get the following output error:

![image](https://user-images.githubusercontent.com/16785357/98041808-d0754600-1e22-11eb-9dc9-ffea7a223792.png)

#### Reproduction steps
```bash
git clone https://github.com/highcharts/highcharts-editor
cd highcharts-editor
npm install
```

#### Editor version
The latest cloned.

**Comment by dgroh** [comment]
It would be nice if you could mention in the docs which version of node you use for development.
We would also appreciate if you could put some milestone for the next release and what should be expected. We plan to use this editor for a project where we have a lot of charts, but we are not sure if this project is still alive as the latest release was over an year ago.

**Comment by machour** [comment]
@dgroh Did you ever got it to compile?

**Comment by dgroh** [comment]
@machour no, we dropped the usage because the editor seems not to be maintained.

**Comment by machour** [comment]
For future readers: 

* Using node v6.17.1, and adding `"less": "2.7.3"` as a dependency in package.json fixed the build for me
* Once standalone.html is open, an error is shown: There's an error with your graph [Object object]. This is because the annotations modules is missing. Adding `<script src="https://code.highcharts.com/modules/annotations.js"></script>` fixes it.

Now stuck with a non-loading advanced panel.
